src/LibraryManager/LibraryNaming/VersionedLibraryNamingScheme.cs

```diff
@@ -1,19 +1,20 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+
 namespace Microsoft.Web.LibraryManager.LibraryNaming
 {
     /// <inheritDoc />
-    class VersionedLibraryNamingScheme : ILibraryNamingScheme
+    internal sealed class VersionedLibraryNamingScheme : ILibraryNamingScheme
     {
         private const char Separator = '@';
 
         /// <summary>
         /// Splits libraryId into name and version using '@' as the split char.
         /// Only the last appearance of '@' is used to split the libraryId.
-        /// Note: If libraryId starts with '@' and has no other occurences of '@',
-        /// the entire libraryId is considered to be the name and version is considered to be
-        /// empty.
+        /// Note: If libraryId starts with '@', a (name + version) substring will be split after the first '/',
```

Comment:
The / is part of the name, but not a delimiter.  For example, there's a big difference `@angular/cli` and `@react/cli`, you can't trim the name of both of these to `cli`.

Reply:
(I think this largely invalidates the commit, so I'm going to hold off looking at the other changes in this commit)

Reply:
Ah, this was addressed later, but the comment wasn't restored.